near-sdk/src/collections/tree_map.rs

```diff
@@ -1777,4 +1777,23 @@ mod tests {
 
         QuickCheck::new().tests(300).quickcheck(prop as Prop);
     }
+
+    #[test]
+    #[cfg(feature = "expensive-debug")]
+    fn test_debug() {
+        let mut rng = rand_xorshift::XorShiftRng::seed_from_u64(4);
+        let mut map = TreeMap::new(b"m");
+        let mut baseline = vec![];
+        for _ in 0..10 {
+            let key = rng.gen::<u64>();
+            let value = rng.gen::<u64>();
+            baseline.push((key, value));
+            map.insert(&key, &value);
+        }
+
+        assert_eq!(
+            format!("{:?}", map),
+            format!("TreeMap {{ root: {:?}, tree: {:?} }}", map.root, map.tree)
+        );
```

Comment:
Again, seems more useful if a few values are added and the actual output is verified, instead of relying on the inner impls which could change and break this format without knowing